openupgradelib/openupgrade.py

```diff
@@ -2683,6 +2688,52 @@ def rename_property(cr, model, old_name, new_name):
     )
 
 
+def update_module_moved_translations(cr, xml_id, old_module, new_module):
+    """Move translations of a specific record from one module to another
+    (usually because a record has been moved from one module to another).
+
+    :param xml_id: a xml record ID
+    :param old_module: previous module of the xml_id
+    :param new_module: new module of the xml_id
+
+    .. deprecated:: 16.0
+       Translations in version 16.0 are in table model, they not depend on module anymore.
+    """
+    if version_info[0] > 15:
+        return
+    cr.execute(
+        """
+        SELECT model, res_id
+        FROM ir_model_data
+        WHERE module = %s AND name = %s
+        """,
+        (
+            old_module,
+            xml_id,
+        ),
```

Comment:
Please reduce the number of lines for something not adding value:

```suggestion
        (old_module, xml_id),
```